.github/workflows/rh-support.yml

```diff
@@ -0,0 +1,29 @@
+name: smoke-tests-against-redhat
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
The new build seems pretty light-weight (probably ~3-5mins run time), but we could consider running the smoke test only on `master` builds in the future, given that we don't really provide support for the Redhat-based image. (No need to change now, let's see how things evolve in the next couple of weeks.. 👍 )

Reply:
👍🏽 That's the plan, the current trigger is just for the initial development.